src/Editor/index.js

```diff
@@ -501,26 +501,28 @@ class Editor extends Component {
         onClick={this.modalHandler.onEditorClick}
         onBlur={this.onWrapperBlur}
       >
-        <div
-          className={ToolbarClass}
-          onMouseDown={this.preventDefault}
-          aria-label="Editor-toolbar"
-          onFocus={this.onToolbarFocus}
-        >
-          {toolbar.options.map((opt, index) => {
-            const Control = Controls[opt];
-            const config = toolbar[opt];
-            return (
-              <Control
-                key={index}
-                {...controlProps}
-                config={config}
-                mention={mention}
-                className="Editor-toolbar-options"
-              />
-            );
-          })}
-        </div>
+        {this.props.showToolbar ?
```

Comment:
We can conditionally render it using && operator. Please refer [this](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator)